CHANGELOG.md

```diff
@@ -3,6 +3,24 @@
 This format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
```

Comment:
```suggestion
```

we need to update the generator to remove this extra blank line, don't want to upset the linters